### DISCUSSION TITLE
Be more explicit about "What to Expect" part of onboarding PR

### DISCUSSION THREAD

**Discussion by TWiStErRob** [renovatebot/renovate#24919] [Suggest an Idea]
### What would you like Renovate to be able to do?

I would like to see what PRs will be created with more specificity. Saying it'll create a Gradle upgrade PR is not enough when the project has about 50 references to many various Gradle versions.
Example: https://github.com/TWiStErRob/net.twisterrob.gradle/pull/228

### Detected Package Files
![image](https://user-images.githubusercontent.com/2906988/186368552-e0ab9ddc-afbd-4be0-8414-65d1af5e6a37.png)

### What to expect
![image](https://user-images.githubusercontent.com/2906988/186368608-9aa3823c-649c-4a81-b3e0-9353b16cccc5.png)

The expectations are not enough to identify if I actually want that upgrade or not. For example I might want to ignore these `/docs/debug/**` folders because they should stay on exactly the version they are. (But after adding the ignore, the version updates stay the same, so either the ignore is not working or there are other places, I just can't know without merging and sifting through 30 PRs)

### If you have any ideas on how this should be implemented, please tell us here.

So, I think a solution here is

1) mention paths that play into each "What to Expect" PR in the detail view.
![image](https://user-images.githubusercontent.com/2906988/186369737-839c8db0-4902-44a5-b05a-676e6b7ed4a7.png)

2) create the whole diff and put it as a code block in the detail view.
```patch
Index: docs/debug/agp343-gradle541/gradle/wrapper/gradle-wrapper.properties
===================================================================
diff
--- a/docs/debug/agp343-gradle541/gradle/wrapper/gradle-wrapper.properties	(revision bd380f95ee676046f45d70dae5a47508bcf55845)
+++ b/docs/debug/agp343-gradle541/gradle/wrapper/gradle-wrapper.properties	(date 1660174343666)
@@ -3,1 +3,1 @@
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.2-all.zip
```

3) [orthogonal to others] Mention the original version(s) where the upgrade comes from (ideally in combination with 1. ), so it is possible to see which versions will change by grepping for a version as text in the codebase.

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
The PR you link to already displays a list of packageFiles detected. For example you can see `/docs/debug/*` files listed there. Wouldn't that be enough for most cases?

**Comment by TWiStErRob** [comment]
Nope, because they're actually ignored... (See related #17379)

So I literally have no idea what else it is trying to upgrade without merging the onboarding and having 30 PRs.

**Comment by rarkins** [comment]
If they are ignored then you know they are not in the PR. i.e. PRs you see in the what to expect can only include package files you see in the what to expect.

**Comment by TWiStErRob** [comment]
So the ignored part was a [red herring](https://github.com/renovatebot/renovate/discussions/17379#discussioncomment-3468182), but the request is still relevant.

If there are multiple places where a library/dependency is used, it's unclear which one will be updated by the PRs listed.

For example as seen above in the screenshots and in [the live report](https://github.com/TWiStErRob/net.twisterrob.gradle/tree/master/docs/debug) there are multiple projects with different version numbers inside the repository. Each of them might be differently out of date and they might be touched by different PRs.
